minikube/bootstrap.sh

```diff
@@ -67,6 +67,17 @@ runCmd() {
   (set -x; $@)
 }
 
+# getGitHubLatestRelease get latest release from GitHub release page
+# Example:
+#   getGitHubLatestRelease "https://github.com/helm/helm/releases/latest"
+getGitHubLatestRelease() {
+  echo $(curl -SsL "$1" | awk '/\/tag\//' \
+    | grep -v no-underline \
+    | head -n 1 | cut -d '"' -f 2 \
+    | awk '{n=split($NF,a,"/");print a[n]}' \
+    | awk 'a !~ $0{print}; {a=$0}')
```

Comment:
@targs08 wow, this is scary. Can we figure out something simpler?

I googled [that one](https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c).

Reply:
@tyranron, I get it from [helm get script](https://github.com/helm/helm/blob/master/scripts/get#L83), but your variant is easier.